app/src/main/java/org/mozilla/focus/engine/CustomContentRequestInterceptor.kt

```diff
@@ -37,7 +37,13 @@ class CustomContentRequestInterceptor(
     }
 
     @Suppress("NestedBlockDepth")
-    override fun onErrorRequest(session: EngineSession, errorCode: Int, uri: String?) {
+    override fun onErrorRequest(session: EngineSession, errorCode: Int, uri: String?): RequestInterceptor.ErrorResponse? {
+
+        fun createErrorResponse(withUri: String, withErrorCode: Int): RequestInterceptor.ErrorResponse {
```

Comment:
nit: remove `with` -> that's not a java/kotlin convention. fwiw, it makes sense in swift where labels are required in some circumstances, becoming part of the method name, and you can supply alternative names for use within the function, e.g. a call in Swift might be `createUri(forString: string)` instead of Java: `createUriForString(string)`

Reply:
I wanted names that didn't shadow the params from the surrounding method, and really struggled to find anything better.  Do you have any suggestions?

Reply:
Huh, I didn't notice that. Maybe `errorResponseUri` and `errorResponseCode`? They provide more accurate specificity than `uri` and `errorCode`. But it's possible someone will refactor those to be "uri" and "errorCode".